docs/elasticsearch/options.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Common Options
+parent: Elasticsearch
+nav_order: 100
+---
+
+# Common options
```

Comment:
So I think we want to break this into two separate pages: Units and Common REST Parameters (or whatever you think for the titles).

Then on the Units page, we mention that the human-readable output parameter exists (and is quite useful) and link to the Common REST Parameters page.